Source/JavaScriptCore/runtime/JSONObject.cpp

```diff
@@ -973,13 +974,13 @@ void FastStringifier::append(JSValue value)
 
     if (value.isInt32()) {
         auto number = value.asInt32();
-        auto length = lengthOfIntegerAsString(number);
-        if (UNLIKELY(!hasRemainingCapacity(length))) {
+        auto* cursor = m_buffer + m_length;
+        auto result = std::to_chars(reinterpret_cast<char*>(cursor), reinterpret_cast<char*>(m_buffer + sizeof(m_buffer)), number);
+        if (UNLIKELY(result.ec == std::errc::value_too_large)) {
             recordBufferFull();
             return;
         }
-        writeIntegerToBuffer(number, &m_buffer[m_length]);
-        m_length += length;
+        m_length += (result.ptr - reinterpret_cast<char*>(cursor));
```

Comment:
I don’t think we need those parentheses.

Reply:
Removed.